### DISCUSSION TITLE
[TabularPredictor] Continue training regardless of the time_limit if necessary

### DISCUSSION THREAD

**Discussion by luizomatias** [autogluon/autogluon#4021] [Q&A] [answered]
I am creating an application with AutoGluon 1.0.0 and setting the time_limit to 7200s (2 hours) with the `presets=high_quality`. With large datasets, it gives an error, for example:

`Not enough time left to train models. Consider specifying a larger time_limit. Time remaining: -8442.07 s
`

I would like to know if it is possible to add some parameter or something so that in these cases where AutoGluon needs more time, it continues training regardless of whether the time_limit has been reached. Is there any way?

**Comment by Omadzze** [comment, marked as answer]
You can just put much bigger time_limit in your case. For example 2 hours time limit is very small for the AutoGluon as it's running multiple models through the training. Put higher time_limit which you can afford and if your model will be trained before the time_limit it will be automatically stop. For example I trained my model with time_limit 9 hours but it stopped after 7 hours as it was fully trained. Hope it helps!
